app/src/main/java/org/greenstand/android/TreeTracker/models/messages/network/RetrofitBuilder.kt

```diff
@@ -0,0 +1,26 @@
+package org.greenstand.android.TreeTracker.models.messages.network
+
+import okhttp3.OkHttpClient
+import okhttp3.logging.HttpLoggingInterceptor
+import retrofit2.Retrofit
+import retrofit2.converter.gson.GsonConverterFactory
+
+class RetrofitBuilder {
+
+    fun create(): Retrofit {
+        return Retrofit.Builder()
+            .client(
+                OkHttpClient.Builder()
+                .addInterceptor(HttpLoggingInterceptor()
+                    .also { it.setLevel(HttpLoggingInterceptor.Level.BODY) })
+                .build())
+            .baseUrl(BASE_ENDPOINT)
```

Comment:
I feel it would be better if we moved the base url to a separate data class in such a way we can have new additional endpoints all in the same class.

Reply:
We can allow multiple endpoints later on.